### DISCUSSION TITLE
install-node.sh customization

### DISCUSSION THREAD

**Discussion by evrynet1** [gravitational/teleport#37827] [Q&A] [answered]
is it possible to adjust/modify the install-node.sh script which is being executed if we Enroll new resources? like adding extra labels etc... to it so they don't need to be added manually after we enrolled the new SSH node to the cluster.

Thank you

**Comment by stevenGravy** [comment, marked as answer]
This is supported for discovery. https://goteleport.com/docs/server-access/guides/ec2-discovery/#step-67-optional-customize-the-default-installer-script. Currently not for that script. 

You can set labels per host id at the auth server. https://goteleport.com/docs/management/admin/labels/#apply-resource-based-labels
